Casks/e/etrecheckpro.rb

```diff
@@ -8,8 +8,8 @@
   homepage "https://etrecheck.com/"
 
   livecheck do
-    url "https://etrecheck.com/en/details.html"
-    regex(/EtreCheckPro\sversion\s(\d+(?:\.\d+)+)/i)
+    url "https://cdn.etrecheck.com/EtreCheckProUpdates.plist"
+    regex(/CFBundleShortVersionString.*?\n.*?(\d+(?:\.\d+)+)/i)
```

Comment:
```suggestion
    strategy :xml do |xml|
      xml.elements["//key[text()='CFBundleShortVersionString']"]&.next_element&.text&.strip
    end
```

We've been using the `Xml` strategy with variations on this approach in other casks recently.

Reply:
Thank you as always #@samford!

I used what we have done with [apparency](https://github.com/homebrew/homebrew-cask/blob/master/Casks/a/apparency.rb), [suspicious-package](https://github.com/homebrew/homebrew-cask/blob/master/Casks/s/suspicious-package.rb) and [archaeology](https://github.com/Homebrew/homebrew-cask/blob/master/Casks/a/archaeology.rb) -- perhaps those could be updated also?

Reply:
> I used what we have done with [apparency](https://github.com/homebrew/homebrew-cask/blob/master/Casks/a/apparency.rb), [suspicious-package](https://github.com/homebrew/homebrew-cask/blob/master/Casks/s/suspicious-package.rb) and [archaeology](https://github.com/Homebrew/homebrew-cask/blob/master/Casks/a/archaeology.rb) -- perhaps those could be updated also?

Yep, I'm working on a PR to standardize the existing `Xml` strategy blocks and I'll be going through plist checks that aren't using `Xml` afterward 👍